src/design/plone/contenttypes/behaviors/evento.py

```diff
@@ -103,10 +68,10 @@ class IEvento(model.Schema):
     )
 
     persone_amministrazione = RelationList(
```

Comment:
come detto anche nell'altra review: se ci sono dei campi utili a più content-type, portiamoli fuori dallo schema e facciamo behavior

Reply:
Sposto solo quelli che non mi incasinano i label per ora, se poi si vede che ne vogliamo mettere altri in fieldset diversi per UX/UI, li spostiamo e si uniforma.

Reply:
non ho capito. Le label si possono fare più generiche, ma ci sono dei campi che sono veramente riutilizzati in diversi content-type ed è uno spreco ricscriverli ogni volta.